eya_def_tools/eya_def_tools/data_models/iea43_wra_data_model.py

```diff
@@ -26,7 +26,10 @@
                 "A wind dataset metadata document according to the IEA "
                 "Wind Task 43 WRA Data Model JSON Schema."
             ),
-            "examples": ["https://foo.com/bar/example_iea43.json"],
+            "examples": [
```

Comment:
This is an unrelated minor refinement. Whilst working on the references to the Power Curve Schema, I realised it is better here to reference a real URL with example data rather than a dummy URL.